packages/rancher-istio-1.7/charts/Chart.yaml

```diff
@@ -1,20 +0,0 @@
-apiVersion: v1
```

Comment:
Why did we delete these files?

Reply:
Clarification offline: We are no longer splitting rancher-istio to 1.7 and 1.8, but we are continuing to rename rancher-istio => rancher-istio-1.8 to facilitate a future release of 1.9